src/index.html

```diff
@@ -2,17 +2,73 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="true"
+    />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/index.css"
```

Comment:
The `href` attribute is correctly pointing to a CSS file (`index.css`). Ensure that your SCSS is compiled into this CSS file, as browsers cannot interpret SCSS directly.